homeassistant/components/incomfort/binary_sensor.py

```diff
@@ -17,6 +22,24 @@
 from .entity import IncomfortEntity
 
 
+@dataclass(frozen=True, kw_only=True)
+class IncomfortBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """Describes Incomfort binary sensor entity."""
+
+    value_key: str
+    extra_state_attributes_fn: Callable[[dict[str, Any]], dict[str, Any]] | None = None
```

Comment:
I suggest that you keep ` | None` for a follow-up PR.
At this stage it is never None.
```suggestion
    extra_state_attributes_fn: Callable[[dict[str, Any]], dict[str, Any]]
```